.gitignore

```diff
@@ -1,4 +1,5 @@
-*.pyc
+/jsonpointer/version.py
```

Comment:
I know this pull request has been sitting for a while, but any chance you can elaborate on why that file would be in ``.gitignore``? For me it would seem logical to have it tracked.